backend/src/services/segmentService.ts

```diff
@@ -157,51 +181,72 @@ export default class SegmentService extends LoggerBase {
   }
 
   async createSubproject(data: SegmentData): Promise<SegmentData> {
+    return SequelizeRepository.withTx(this.options, async (tx) => {
+      const qx = SequelizeRepository.getQueryExecutor(this.options, tx)
+      return this.createSubprojectInternal(data, qx, tx)
+    })
+  }
+
+  async createSubprojectInternal(
+    data: SegmentData,
+    qx: QueryExecutor,
+    transaction: Transaction,
+  ): Promise<SegmentData> {
     if (!data.parentSlug) {
       throw new Error('Missing parentSlug. Subprojects must belong to a project.')
     }
 
     if (!data.grandparentSlug) {
       throw new Error('Missing grandparentSlug. Subprojects must belong to a project group.')
     }
-    const transaction = await SequelizeRepository.createTransaction(this.options)
 
-    try {
-      const segmentRepository = new SegmentRepository({
-        ...this.options,
-        transaction,
-      })
+    const segmentRepository = new SegmentRepository({ ...this.options, transaction })
+    const collectionService = new CollectionService({ ...this.options, transaction })
 
-      const parent = await segmentRepository.findBySlug(data.parentSlug, SegmentLevel.PROJECT)
+    const parent = await segmentRepository.findBySlug(data.parentSlug, SegmentLevel.PROJECT)
+    if (!parent) {
+      throw new Error(`Project ${data.parentSlug} does not exist.`)
+    }
 
-      if (parent === null) {
-        throw new Error(`Project ${data.parentSlug} does not exist.`)
-      }
-      if (parent.isLF === false) data.slug = validateNonLfSlug(data.slug)
+    if (parent.isLF === false) {
+      data.slug = validateNonLfSlug(data.slug)
+    }
 
-      const grandparent = await segmentRepository.findBySlug(
-        data.grandparentSlug,
-        SegmentLevel.PROJECT_GROUP,
-      )
+    const grandparent = await segmentRepository.findBySlug(
+      data.grandparentSlug,
+      SegmentLevel.PROJECT_GROUP,
+    )
+    if (!grandparent) {
+      throw new Error(`Project group ${data.grandparentSlug} does not exist.`)
+    }
 
-      if (grandparent === null) {
-        throw new Error(`Project group ${data.parentSlug} does not exist.`)
-      }
+    const subproject = await segmentRepository.create({
+      ...data,
+      parentId: parent.id,
+      grandparentId: grandparent.id,
+      isLF: parent.isLF,
+    })
 
-      const subproject = await segmentRepository.create({
-        ...data,
-        parentId: parent.id,
-        grandparentId: grandparent.id,
-        isLF: parent.isLF,
-      })
+    const collections = await findBySlug(qx, data.grandparentSlug)
 
-      await SequelizeRepository.commitTransaction(transaction)
+    const [existingProject] = await collectionService.findInsightsProjectsByName(subproject.name)
 
```

Comment:
I believe that on line 233 we are setting starred to true by default:
```
if (project.collections) {
        await connectProjectsAndCollections(
          qx,
          project.collections.map((c) => ({
            insightsProjectId: createdProject.id,
            collectionId: c,
            starred: true,
          })),
        )
      }
```

By default this should be set to false